packages/table/test/tableTests.tsx

```diff
@@ -562,13 +566,15 @@ describe("<Table>", () => {
             document.body.appendChild(containerElement);
 
             const tableComponent = ReactDOM.render(
-                <Table
-                    numRows={LARGE_NUM_ROWS}
-                    numFrozenColumns={NUM_FROZEN_COLUMNS}
-                    numFrozenRows={NUM_FROZEN_ROWS}
-                >
-                    {renderColumns({}, LARGE_NUM_COLUMNS)}
-                </Table>,
+                <div style={{ height: CONTAINER_HEIGHT, width: CONTAINER_WIDTH }}>
```

Comment:
That was the point of this PR actually: to fix that.

BEFORE (buggy):
![image](https://user-images.githubusercontent.com/443450/29589826-bbaa3576-874b-11e7-96b5-d5a9239c41b4.png)


AFTER (fluid height):
![image](https://user-images.githubusercontent.com/443450/29589834-c55e06ba-874b-11e7-8391-637ee16ea5ed.png)